lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -67,6 +67,10 @@
     description:
       - opaque blob of data which is made available to the ec2 instance
     required: false
+  user_data_path:
+    description:
+      - Path to the file that contains userdata for the ec2 instances
+    required: false
```

Comment:
Please add the following line directly below `required: false`
`version_added: "2.3"`